images/prometheus/configs/latest.pushgateway.apko.yaml

```diff
@@ -0,0 +1,23 @@
+contents:
+  packages:
+    - prometheus-pushgateway
+
+entrypoint:
+  command: /usr/bin/pushgateway
+
+accounts:
+  groups:
+    - groupname: nonroot
+      gid: 65532
+  users:
+    - username: nonroot
+      uid: 65532
+      gid: 65532
+  run-as: 65532
```

Comment:
I'm a little worried this might not be wai. Can we change this to match upstream just in case?

https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-pushgateway/values.yaml#L186

Reply:
Done!